docs/docs-ref-autogen/excel/toc.yml

```diff
@@ -621,3 +645,7 @@ items:
             uid: ExcelScript!ExcelScript.WorksheetProtectionOptions:interface
           - name: WorksheetSearchCriteria
             uid: ExcelScript!ExcelScript.WorksheetSearchCriteria:interface
+          - name: EmailAttachment
```

Comment:
There's some bug here where the new APIs aren't showing up under the "Global" namespace. Probably in the tools, but might be in the d.ts. I'll investigate soon.

I do want to know how these are used in Scripts. Do we have sample code?